.github/workflows/daily.yml

```
@@ -19,6 +19,57 @@ on:
         default: 'master'
 
 jobs:
+  tests-with-elle:
+    runs-on: ubuntu-latest
+    if: |
+      (github.event_name == 'workflow_dispatch' ||
+      (github.event_name == 'schedule' && github.repository == 'redislabs/redisraft')) && !contains(github.event.inputs.skipjobs, 'elle')
+    timeout-minutes: 14400
+    steps:
+    - name: prep
+      if: github.event_name == 'workflow_dispatch'
+      run: |
+        echo "GITHUB_REPOSITORY=${{github.event.inputs.use_repo}}" >> $GITHUB_ENV
+        echo "GITHUB_HEAD_REF=${{github.event.inputs.use_git_ref}}" >> $GITHUB_ENV
+    - uses: actions/checkout@v2
+      with:
+        repository: ${{ env.GITHUB_REPOSITORY }}
+        ref: ${{ env.GITHUB_HEAD_REF }}
+    - name: Install build dependencies
+      run: sudo apt-get install -y build-essential autoconf automake libtool cmake lcov leiningen default-jdk-headless
+    - name: Build
+      run: |
+        mkdir build && cd build
+        cmake .. -DPYTEST_OPTS="--redis-executable=redis/src/redis-server --elle-cli=elle-cli/target/elle-cli-0.1.4-standalone.jar --elle_threads=3 -v"
```

```suggestion
        cmake .. -DPYTEST_OPTS="--redis-executable=redis/src/redis-server --elle-cli=elle-cli/target/elle-cli-0.1.4-standalone.jar --elle-threads=3 -v"
```

Please run daily with the above fix to verify we don't break the build and it actually runs `elle`. 

You can run on Github by using this branch. 
On your fork, `Actions` -> `Daily` -> `Run Workflow` -> `Use workflow from`: select your branch, then fill out the boxes. 

Not necessary right now but to verify we actually run Elle, we can count `success` response count and assert it is greater than zero. Also, counting `asking` and `moved` responses and printing to the log at the end similar to Jepsen would be nice.